sdk/loadtesting/azure-developer-loadtesting/tsp-location.yaml

```diff
@@ -1,4 +1,4 @@
-directory: specification/loadtestservice/resource-manager/Microsoft.LoadTestService/loadtesting
-commit: 05584a1019e75159b0dc70a6751afaa2c77868e6
+directory: specification/loadtestservice/data-plane/loadtesting
```

Comment:
This was due to my mistake in https://github.com/Azure/azure-sdk-for-java/pull/46791

When update `commit` (and the follow-up to update `directory` due to specs folder refactor), I mistakenly point the directory to control-plane.